Jenkinsfile

```diff
@@ -1,3 +1,4 @@
+// TODO replace all this with: buildPlugin(configurations: buildPlugin.recommendedConfigurations()) (yes, Scripted, so what)
```

Comment:
Submit a PR that does the replacement or file a JIRA for this task.
This does not belong in a TODO in this file and certainly not in this PR.